sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ConfluentSchemaRegistryDeserializerProvider.java

```diff
@@ -87,6 +87,21 @@
         version);
```

Comment:
To avoid code duplication, can we just return `of(schemaRegistryUrl, subject, version, null)` instead here?

Reply:
@aromanenko-dev  Yes, makes sense, this is now done, PTAL. thanks!